dashboard/craco.config.js

```diff
@@ -0,0 +1,19 @@
+const CracoLessPlugin = require("craco-less");
+
+module.exports = {
+  plugins: [
+    {
+      plugin: CracoLessPlugin,
+      options: {
+        lessLoaderOptions: {
+          lessOptions: {
+            modifyVars: {
+              "@primary-color": "#1DA57A",
```

Comment:
Just curious: what drove this particular color choice?

Reply:
Just some random color :grinning: